api/alembic/versions/cb626d81dec9_load_fuel_types.py

```diff
@@ -26,7 +26,7 @@
 
 def upgrade():
     # Iterate through the fuel types and insert them.
-    for fuel_type_id, geom in fuel_type_iterator():
+    for fuel_type_id, geom in fuel_type_iterator('fbp2021.tif'):
```

Comment:
Changing old migrations is probably not the best habit...

Reply:
I think this is the only way to make it backwards compatible, thanks for doing that! Hopefully this will never need to run in prod :)

Reply:
If you don't want to change this migration you could make the parameter optional and have a default value of 'fbp2021.tif'.

Reply:
I did consider that, opted against setting the default to an out of date fuel grid